code/style.css

```diff
@@ -1,7 +1,497 @@
+
+/* VIDEO CODE */
+
+.video-container {
+  height: 100%;
+  width: 100%;
+  overflow: hidden;
+  position: relative;
+  z-index: -1;
+  top: -39%;
+  filter: sepia(1) hue-rotate(230deg);
+  opacity: 0.2;
+}
+
+video {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translateX(-50%) translateY(-50%);
+}
+
+
+/* The hero class contains everything about the hero section (between nav and main) */
+.header-content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: relative;
+  top: 35%;
+  left: 0%;
+}
+
+.header-content img {
+  max-width: 400px;
+}
+
+.header-cta > button {
+  margin: 0 8px 0 8px;
+}
+
+button {
+  border-radius: 100px;
+  padding: 12px 32px;
+  cursor: pointer;
+
+}
+
+.button-primary {
+  border: none;
+  color: #1A2B34;
+}
+
+.button-secondary {
+  border: 1px solid;
+  background-color: rgba(0,0,0,0);
+}
+
+button > p {
+  margin: 0;
+  font-weight: 700;
+}
+
+
+/* ====== TYPOGRAPHY ======= */
+
+p, span, h1, h2, h3 {
+  -webkit-font-smoothing: antialiased;
+}
+
+h1, h2, h3 {
+  font-family: 'Roboto Mono', monospace;
+  font-style: normal;
+  font-weight: bold;
+  margin-top: 0;
+}
+
+h1 {
+  font-size: 42px;
+  line-height: 55px;
+}
+
+h2 {
+  font-size: 30px;
+  line-height: 40px;
+  margin-bottom: 16px;
+}
+
+.nav-links > a {
+  margin-left: 24px;
+}
+
+a {
+  font-family: 'Lexend Peta', sans-serif;
+  text-decoration: none;
+  color: rgba(255,255,255,0.7);
+  font-size: 14px;
+  text-transform: uppercase;
+}
+a:hover {
+  color: rgba(255,255,255,0.85);
+}
+
+.emphasise {
+  font-style: italic;
+}
+
+
+/* select h2 in a class card-full */
+.card-full > h2  {
+  margin: 0 auto 16px 16px;
+}
+
+.card-full {
+}
+
+h3 {
+  font-size: 24px;
+  line-height: 32px;
+}
+
+p {
+  font-family: 'Roboto Slab', serif; 
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  
+}
+
+.p-small {
+  font-family: 'Roboto Slab', serif; 
+  font-size: 16px;
+}
+
+.smaller {
+  font-size: 14px;
+  color: rgba(255,255,255,0.7);
+}
+
+.smaller a {
+  font-family: 'Roboto Slab', serif; 
+  color: #5DC3CA;
+  text-transform: none;
+  font-size: 14px;
+}
+
+/* ===== form input styling ===== */
+
+label {
+  margin: 0 0 8px 16px;
+}
+
+.bold {
+  font-weight: 700;
+}
+
+input[type=text], input[type=email] {
+  border-radius: 24px;
+  border: 1px solid rgba(0,0,0,0.20);
+  padding: 10px 24px;
+
+  font-family: 'Roboto Slab', serif; 
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  margin-bottom: 24px;
 }
+
+input[type=text]:focus, input[type=email]:focus {
+  border: 1px solid #BF85BF;
+  outline-width: 0;
+  box-sizing: border-box;
+  box-shadow: inset 0 0 0px 2px #BF85BF;
+}
+
+.slider {
+  -webkit-appearance: none;
+  appearance: none;
+  height: 8px;
+  background-color: rgba(0,0,0,0.11);
+  border-radius: 100px;
+  width: 80%;
+}
+
+.slider:focus {
+  outline-width: 0;
```

Comment:
This is accessibility style we shouldn't get rid of, that's an antipattern. You can always change default style, but not hide it completely